app/controllers/api/v0/pages_controller.rb

```diff
@@ -1,8 +1,10 @@
 class Api::V0::PagesController < Api::V0::ApiController
   def index
     query = page_collection
+    sorting = sortation(params[:sort])
     paging = pagination(query)
     pages = query.limit(paging[:chunk_size]).offset(paging[:offset])
+    pages = pages.order(sorting) unless sorting.nil?
```

Comment:
I think the sorting may need to be added to `query`, not `pages` here, since the workaround for an ActiveRecord bug immediately below this code uses `query` instead of `pages` to build the final SQL query.

This also definitely needs tests, since that would reveal that the built-in sorting for the [`lightweight_query` functionality](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/app/controllers/api/v0/pages_controller.rb#L129-L135) for this controller will conflict with and override any sorting provided here. I think you’ll need to fix that, too.